src/components/ThreeDotsMenu/index.js

```diff
@@ -61,66 +59,58 @@ const defaultProps = {
     },
```

Comment:
looks like the conflict is from here https://github.com/Expensify/App/commit/742cea293eac00a0522bf249fb7427f04c01d7bf#diff-cbdafcfd10fb7026d43c5578f29ca1a81a942b19ca3736fcf66795a01f9cfe91

So just need to pull `main`, re-apply your changes and then updated the defaultProp of `anchorAlignment` with
```
vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.TOP, // we assume that popover menu opens below the button, anchor is at TOP
```